clients/unshaded/src/main/java/tachyon/client/file/options/CreateOptions.java

```diff
@@ -143,6 +143,15 @@ public UnderStorageType getUnderStorageType() {
   }
 
   /**
+   * @return the name : value pairs for all the fields
+   */
+  @Override
+  public String toString() {
+    return "CreateTOptions{mBlockSizeBytes=" + mBlockSizeBytes + ",mRecursive=" + mRecursive
```

Comment:
@calvinjia Agreed on readability with addition of spaces, I can make a helper function toStringBase() to call super.toString() specifically for tests, plus replace string literals with toStringHelper. How does that sound?